src/util-inl.h

```diff
@@ -198,6 +198,31 @@ TypeName* Unwrap(v8::Local<v8::Object> object) {
   return static_cast<TypeName*>(pointer);
 }
 
+template <typename TypeName>
+void SwapBytes(TypeName* dst,
+               const TypeName* src,
+               size_t buflen) {
+  // No types with size > 8 or odd sizes (not 2-byte aligned) permitted.
+  const size_t t_size = sizeof(TypeName);
+  static_assert(sizeof(TypeName) <= 8
+  && (sizeof(TypeName) & 0x1) == 0, "Unsupported type");
```

Comment:
No linter errors here (or elsewhere in the code), at least from my `make lint` tests.